src/vcastCommandRunner.ts

```diff
@@ -62,7 +62,9 @@ function processExceptionFromExecuteCommand(
 
   // 99 is a warning, like a mismatch opening the environment
   if (error && error.status == 99) {
-    commandStatus.stdout = error.stdout.toString();
+    let stdoutString =  error.stdout.toString();
+    // Needed because of the annoying version miss-match message from vcast
+    commandStatus.stdout = stdoutString.split ("ACTUAL-DATA", 2)[1].trim ()
```

Comment:
In the past, when I've "broken" things, we get the Python traceback in the message area.

Is this going to break that?

Reply:
The py utilities ALWAYS "print" ACTUAL-DATA as the first thing in the main to make the processing in TS easy.
So any crash etc will be after the ACTUAL*

Reply:
`testEditorInterface.py` prints it _after_ doing its processing

Reply:
Good point we can  move them to global scope or the start of the main for consistency

Reply:
Start of main would match what's in `vTestInterface.py`

Reply:
But not crc32.py :)  ... done